package.json

```diff
@@ -56,13 +56,14 @@
     "gitbook-plugin-sitemap": "1.2.0",
     "jest": "23.4.0",
     "jest-junit": "5.1.0",
-    "lodash": "4.17.10",
+    "lodash.merge": "^4.6.1",
```

Comment:
We've marked this as a `devDependency`, but because we `import` it into `BaseTool.js`, it should be under `dependencies`

Reply:
I have removed both dependencies.